exercises/accumulate/README.md

```diff
@@ -29,10 +29,10 @@ Solve this one yourself using other basic tools instead.
 
 To run the tests run the command `go test` from within the exercise directory.
 
-If the test suite contains benchmarks, you can run these with the `--bench` and `--benchmem`
-flags:
+If the test suite contains benchmarks, you can run these with the `-bench`
+flag:
```

Comment:
Did you notice that the differences shown dropped some standard lines for the Go READMEs here? The standard lines come from [here](https://github.com/exercism/go/blob/master/config/exercise_readme.go.tmpl#L11-L14).
And, the README.md files are generated using a tool called configlet, that gathers pieces from standard places, track specific,  and also from the .meta directory (if applicable) under each exercise.  For a few exercises, the base content of the README.md comes from a .meta/description.yml file under the exercise directory. For example, see exercises/yacht; and in those cases, the .meta directory must also include a copy of the metadata.yml from the problem-specifications/exercises/.../metadata.yml (without change).
First, clone the problem-specifications repo, at the same level, directory-wise, as your local exercism/go repo.
Use exercises/accumulate/description.md from that repo to make your adjustments (numbers vs. strings) and put the result in exercises/accumulate/.meta/description.md; copy the metadata.yml for yacht from problem-specifications/exercises/accumulate/metadata.yml into the new accumulate/.meta directory
Then get a up-to-date copy of the configlet tool:
```
./bin/fetch-configlet
```
Then, run configlet:
```
./bin/configlet help generate
./bin/configlet generate --only accumulate
```
The README.md generated for accumulate should show only the specific changes you made for the strings vs. numbers and that text would be in the exercises/accumulate/.meta/description.md.